.circleci/config.yml

```diff
@@ -154,10 +159,28 @@ jobs:
 
       - setup_remote_docker
 
+      - run:
+          name: Load image layer cache
+          command: |
+            set +o pipefail
+            docker load -i /caches/deb-builder.tar |true
```

Comment:
Why `pipefail` to `true`. Won't that just exit the same as if you ran the command directly?

Reply:
hah honestly i just copied and pasted the logic that was there before that wasnt written by me. Im not opposed to tearing it out as I dont have any justification for it being there.